pulsar/consumer_impl.go

```diff
@@ -364,3 +356,21 @@ func toProtoInitialPosition(p SubscriptionInitialPosition) pb.CommandSubscribe_I
 
 	return pb.CommandSubscribe_Latest
 }
+
+func (c *consumer) messageID(msgID MessageID) (*messageID, bool) {
```

Comment:
maybe call this validateMessageID?

Reply:
I though this before, but this function will convert MessageID(uppercase M) to messageID(lowercase m), so any function name like XXXMessageID, will cause confusion...